create_database.sql

```diff
@@ -298,6 +298,8 @@ CREATE TABLE `instrument` (
   `created_on` datetime DEFAULT NULL,
   `updated_by` varchar(45) DEFAULT NULL,
   `updated_on` datetime DEFAULT NULL,
+  CONSTRAINT `instrument_serial_UNIQUE` UNIQUE KEY `instrument_serial_UNIQUE`(`instrument_serial`),
+  CONSTRAINT `participant_id_UNIQUE` UNIQUE KEY `participant_id_UNIQUE`(`participant_id`),
```

Comment:
This is incorrect and will only allow one record per participant_id and only one record per instrument_serial.

We need a unique constraint on participant_id and instrument_serial so that there can be only one of each instrument serial number per participant.